staging/src/k8s.io/component-base/logs/registry.go

```diff
@@ -21,6 +21,11 @@ import (
 	"sync"
 
 	"github.com/go-logr/logr"
+	json "k8s.io/component-base/logs/json"
+)
+
+const (
+	jsonLogFormat = "json"
```

Comment:
nit: what does this constant add?

Reply:
I think the  json log format string key should be fixed.